src/GPSs.cpp

```diff
@@ -0,0 +1,53 @@
+#include "GPSs.h"
+#include <Adafruit_GPS.h>
+#include <FlexCAN_T4.h>
+
+GPSs::GPSs(int id, HardwareSerial* serialIn) {
+    this->id = id;
+    this->wGPS = Adafruit_GPS(serialIn);
+}
+
+void GPSs::begin() {
+    this->wGPS.begin(9600);
+    this->wGPS.sendCommand(PMTK_API_SET_FIX_CTL_5HZ);
+    this->wGPS.sendCommand(PMTK_ENABLE_WAAS);
+}
+
+Health GPSs::healthCheck() {
+    unsigned long startCheck = millis();
+
+    while(millis() - startCheck <= 500) {
```

Comment:
I'm assuming this is so there is at *most* a 500ms delay in a health check response in the case that wGPS.read() hangs or waits?